cmd/root.go

```diff
@@ -117,12 +117,15 @@ or ./scorecard --{npm,pypi,rubgems}=<package_name> [--checks=check1,...] [--show
 
 		enabledChecks := checker.CheckNameToFnMap{}
 		if len(checksToRun) != 0 {
+		loop:
 			for _, checkToRun := range checksToRun {
-				if checkFn, ok := checks.AllChecks[checkToRun]; ok {
-					enabledChecks[checkToRun] = checkFn
-				} else {
-					log.Fatalf("Invalid check: %s", checkToRun)
+				for key, checkFn := range checks.AllChecks {
```

Comment:
Instead of `continue loop`, put the nested `for` inside a fn:

```
for _, checkToRun := range checksToRun {
  if b := parseCheckName(checkToRun, &enabledChecks); b == false {
    log.Fatalf("Invalid check: %s", checkToRun)
  }
}
```